control-plane/plugin/src/resources/pool.rs

```diff
@@ -133,7 +166,17 @@ impl ListWithArgs for Pools {
             },
             None => true,
         });
-        utils::print_table(output, pools);
+
+        let pools_display = PoolDisplayLabels::new_pools(pools.clone(), args.show_labels());
```

Comment:
Doesn't seem to be used afterwards
```suggestion
        let pools_display = PoolDisplayLabels::new_pools(pools, args.show_labels());
```